packages/eslint-plugin-percolate/lib/rules/test-suite-name.js

```diff
@@ -0,0 +1,40 @@
+const { basename, dirname, extname, join, relative } = require('path')
+
+module.exports = {
+    meta: {
+        docs: {
+            description: 'Enforces a test suite name matches the filename',
+            category: 'Best Practices',
+            recommended: true,
+        },
+        fixable: true,
+    },
+
+    create: function(context) {
+        const { basePaths = ['/'] } = context.options[0] || {}
+        return {
+            ExpressionStatement: node => {
+                if (node.expression.type !== 'CallExpression') return
+                if (node.expression.callee.name !== 'describe') return
+                if (node.expression.arguments.length < 1) return
+
+                const ancestors = context.getAncestors()
+                if (ancestors[ancestors.length - 1].type !== 'Program') return
+
+                const filename = context.getFilename()
+                const basePath = basePaths.find(path => filename.indexOf(path) === 0)
+                const relativePath = basePath
+                    ? relative(basePath, join(dirname(filename), basename(filename, extname(filename))))
```

Comment:
Looks normal to me. Anyway, since it has a fix option, why would you ever need to type the name anyway?